internal/npm_install/test/golden/node_modules/zone.js/BUILD.bazel.golden

```diff
@@ -1,15 +1,18 @@
 
 package(default_visibility = ["//visibility:public"])
 load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
-filegroup(
+load("@build_bazel_rules_nodejs//internal/npm_install:node_module_library.bzl", "node_module_library")
+node_module_library(
     name = "zone.js__pkg",
     srcs = [
         ":zone.js__files",
+    ],
```

Comment:
nit: you could reduce the delta by not writing empty deps

Reply:
done